google/web-stories-wp
# Layers Panel: Layer Action Scrim Container

## Context
Part of design updates for layers panel.


## Summary
Adds a container that will house the layer actions (soon to be built). This container appears on hover. Right now there are no actions other than the disabled `lock/unlock` action on the Background element.


## Relevant Technical Choices
Went with a css predominant approach to minimize having to hold button state in js.

Also went with a box-shadow implementation for the scrim instead of an element with a gradient to simplify positioning and nodes.


## To-do



## User-facing changes
NA (Small right aligned scrim appears on hover in the layers panel now)


## Testing Instructions
Interact with the layers panel and see that there's now a little right aligned scrim that appears on hover. Other than that, just make sure nothing had any regressions.





This PR can be tested by following these steps:

1.


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #9140
